README.md

```diff
@@ -48,159 +47,156 @@ limitations under the License.
     </a>
 </p>
 
-## Overview
+<h3> DeepSparse is a deep learning inference engine for running super-fast sparse models. 🚀🚀🚀</h3>
```

Comment:
We should keep the original tagline for now as these changes require a larger consensus. The final approved copy would need to be updated not just in our Naming Brief but in all the places that utilize this asset.

Reply:
ok will keep original for now.